var/spack/repos/builtin/packages/miopen-hip/package.py

```diff
@@ -174,14 +197,21 @@ def cmake_args(self):
             args.append(self.define("MIOPEN_USE_MLIR", "OFF"))
         if self.spec.satisfies("@5.7.0:"):
             args.append(self.define("MIOPEN_ENABLE_AI_IMMED_MODE_FALLBACK", "OFF"))
-        args.append(
-            "-DNLOHMANN_JSON_INCLUDE={0}".format(self.spec["nlohmann-json"].prefix.include)
-        )
-        if self.spec.satisfies("@6.0.0:"):
+        if self.spec.satisfies("@6.0.0:6.1"):
```

Comment:
FWIW You could simplify this:

```suggestion
        if self.spec.satisfies("@6:6.1"):
```